docs/src/main/paradox/worker.md

```diff
@@ -0,0 +1,17 @@
+# The Worker Nodes
+
+When the worker receives work from the master it delegates the actual processing to a child actor, `WorkExecutor`, to keep the worker responsive while executing the work.
```

Comment:
Suggest replacing four paragraphs with:

'Worker' actors and the 'Master' actor interact as follows:
1. 'Worker' actors register with the 'Master' so the master knows they are available and ready to take on work.
2.  When the 'Master' actor has work, it sends a 'WorkIsReady' message to all workers it things are not busy. 
3. 'Worker' actors that are 'idle' will reply to the 'Master' with a 'WorkerRequestsWork' mesage. 
4. The 'Master' picks the first reply and assigns the work to that worker.  This achieves back pressure because the 'Master' does not push work on workers that are already busy and overwhelm their inboxes.
5. When the worker receives work from the master, it delegates the actual processing to a child actor, `WorkExecutor`. This allows the worker to be responsive while its child executes the work.